diff -Naur steamos-compositor-1.29.old/usr/bin/steamos-session steamos-compositor-1.29/usr/bin/steamos-session
--- steamos-compositor-1.29.old/usr/bin/steamos-session	2015-07-31 14:08:17.021634896 +0200
+++ steamos-compositor-1.29/usr/bin/steamos-session	2015-07-31 14:08:34.853249268 +0200
@@ -8,7 +8,7 @@
 
 /usr/bin/steamos/set_hd_mode.sh >> $HOME/set_hd_mode.log
 
-export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libmodeswitch_inhibitor.so:/usr/lib/i386-linux-gnu/libmodeswitch_inhibitor.so
+export LD_PRELOAD=/usr/lib64/libmodeswitch_inhibitor.so.0.0.0:/usr/lib/libmodeswitch_inhibitor.so.0.0.0
 
 # Disable DPMS and screen blanking for now; it doesn't know about controller
 # or Steam remote control events right now
